packages/litqa/src/aviary/envs/litqa/task.py

```diff
@@ -12,7 +12,7 @@
 from paperqa.settings import Settings
 from paperqa.types import DocDetails
 
-from aviary.core import (
+from aviary.core import (  # type: ignore[attr-defined]
```

Comment:
I looked into this, how strange

Will figure this one out later 😆 let's just leave for now